middlewares.go

```diff
@@ -58,7 +58,25 @@ func JwtMiddleware(key string) negroni.Handler {
 		},
 		SigningMethod: jwt.SigningMethodHS256,
 	})
-	return negroni.HandlerFunc(jwtMiddleware.HandlerWithNext)
+	// return negroni.HandlerFunc(jwtMiddleware.HandlerWithNext)
+	return negroni.HandlerFunc(func(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
+		match, err := MatchURL(r.URL.String())
+		if err != nil {
+			http.Error(w, fmt.Sprintf(`{"error": "%v"}`, err), http.StatusInternalServerError)
+			return
+		}
+		if match {
+			next(w, r)
+			return
+		}
+		err = jwtMiddleware.CheckJWT(w, r)
+		if err != nil {
+			log.Println("check jwt error", err.Error())
+			w.Write([]byte(fmt.Sprintf(`{"error": "%v"}`, err.Error())))
```

Comment:
do not need to write error message because handlerSet middleware already write errors in response